shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -58,7 +54,9 @@ private static void printMetaMasterInfo(String spaces) {
               MasterInfoField.UP_TIME_MS, MasterInfoField.VERSION,
               MasterInfoField.SAFE_MODE));
       MasterInfo masterInfo = client.getMasterInfo(masterInfoFilter);
-
+      if (masterInfo == null) {
+        throw new IOException("Cannot get meta master info from meta master client");
```

Comment:
does the client return null when it fails to get meta master info?

Reply:
Even when we cannot get any values, the client.getMasterInfo will return an empty MasterInfo(All fields have java type default value, like null or 0).

Thanks for pointing out that, I will change it.

Reply:
Do we need to check all the field values? If they are not default value, then we print out.

Like 
```java
String masterAddress = masterInfo.getMasterAddress();
if (masterAddress == null) {
  throw new IOException("Cannot get master address from meta master client");
}
print("Master Address: " + masterAddress);
```
The code will look long if we do it for all the fields.

Reply:
@aaudiber do we need to check all the fields before printing out.
Otherwise, if the Thrift server side does not work well, we will print out all the info with default values which may look strange for users.